src/script/message/MessageSender.js

```diff
@@ -0,0 +1,33 @@
+/*
+ * Wire
+ * Copyright (C) 2019 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import PromiseQueue from 'utils/PromiseQueue';
+
+export class MessageSender {
```

Comment:
I have great plans for this `MessageSender`. In the future it will be the only class knowing how to send a message on Wire. 
Its reponsabilities will include:
- encrypting the messages 
- handling 412 responses from the backend and warn the app that something needs to be done. 

Right now it's mostly an empty shell around the `PromiseQueue` but it will evolve with some followup PR